lexicons/tools/ozone/hosting/getAccountHistory.json

```diff
@@ -0,0 +1,106 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.hosting.getAccountHistory",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get account history, e.g. log of updated email addresses or other identity information.",
+      "parameters": {
+        "type": "params",
+        "required": ["did"],
+        "properties": {
+          "did": { "type": "string", "format": "did" },
+          "event": {
```

Comment:
do we want to use `events` here, I believe we've always used plural for arrays?

Reply:
Yup, good call 👍